Signal/src/views/MarqueeLabel.swift

```diff
@@ -0,0 +1,1855 @@
+//  Grabbed from: https://github.com/cbpowell/MarqueeLabel-Swift/blob/cd331f3cfc3f9d7114ffa5aa4f243f1d5eda9d0d/Classes/MarqueeLabel.swift
+//  License: MIT License
+//
+//  MarqueeLabel.swift
+//
+//  Created by Charles Powell on 8/6/14.
+//  Copyright (c) 2015 Charles Powell. All rights reserved.
+//
```

Comment:
Won't this header get rewritten by the precommit script?

Reply:
Our precommit script already ignores source inside directories named "ThirdParty".  We should move this to the ThirdParty dir.